### ISSUE TITLE
Gutenberg: Multiple pullquotes on one page: colour issues

### ISSUE THREAD

**Issue by sophiegyo** [Automattic/wp-calypso#30924]
TL;DR: When a page has more than one pullquote in solid style colour, and the colours are changed after the page is published, all pullquotes after the first remain stuck on #0073aa as a background colour.

I have not tested this in wp-admin!
I did test on jurassic.ninja and couldn't reproduce, it really seems to be Calypso-specific.

#### Steps to reproduce
1. Starting at URL:  https://wordpress.com/pages/ - feel free to use my testing site gutenlumos (it's a free site, so only ends in .wordpress.com)
2. Create a new page
3. Add a minimum of two pullquotes, with solid background colour
4. Publish and check the page
5. Return to edit the page in Calypso, and change the colour of any but the first pullquote
6. Publish and check the page anew

#### What I expected
That the other pullquote(s) would change background colour appropriately

#### What happened instead
All pullquotes after the first remain stuck with this background colour: #0073aa
It looks like the additional pullquotes just keep adding background colour classes.

#### Browser / OS version
Firefox recent - will grab the browser version after I restart so it actually tells me.
OS X 10.14 

#### Screenshot / Video
https://cld.wthms.co/iOBJi4 - updating an existing pullquote to a new colour
https://cld.wthms.co/l0pO3p - pullquote has been updated from one colour to another, and has an extra css class for the background colour.

#### Context / Source
#manual-testing
I noticed this when exploring how I could recreate the Twenty Nineteen demo.

**Comment by arunsathiya** [comment]
I am not able to reproduce this on https://wordpress.com/block-editor/post. Could this be theme-specific?

| <img width="759" alt="screenshot 2019-02-21 at 17 49 06" src="https://user-images.githubusercontent.com/18581859/53168630-1f545b00-3601-11e9-8266-c243ae2ae32a.png"> | <img width="661" alt="screenshot 2019-02-21 at 17 49 12" src="https://user-images.githubusercontent.com/18581859/53168631-1f545b00-3601-11e9-94c0-53058867f8fc.png"> |
|------|------|
| <img width="720" alt="screenshot 2019-02-21 at 17 49 33" src="https://user-images.githubusercontent.com/18581859/53168633-1fecf180-3601-11e9-8243-30ebcc620a1b.png"> | <img width="630" alt="screenshot 2019-02-21 at 17 49 26" src="https://user-images.githubusercontent.com/18581859/53168632-1f545b00-3601-11e9-916c-1db965f835ba.png"> |

First column of live view, and second of editor. Hemingway rewritten.

**Comment by arunsathiya** [comment]
Actually, I see what you mean on Twenty Nineteen. 

| <img width="634" alt="screenshot 2019-02-21 at 17 55 03" src="https://user-images.githubusercontent.com/18581859/53168953-f1bbe180-3601-11e9-8f0a-9c628db770c8.png"> | <img width="968" alt="screenshot 2019-02-21 at 17 55 08" src="https://user-images.githubusercontent.com/18581859/53168956-f1bbe180-3601-11e9-94cd-5106017b4cc1.png"> |
|------|------|
| <img width="623" alt="screenshot 2019-02-21 at 17 55 33" src="https://user-images.githubusercontent.com/18581859/53168958-f2547800-3601-11e9-9be0-5c19cb241a8c.png"> | <img width="968" alt="screenshot 2019-02-21 at 17 55 37" src="https://user-images.githubusercontent.com/18581859/53168959-f2547800-3601-11e9-8aba-ae1600532c3f.png"> |

But, it is not reproducible on a .org site. Pressable host, if that's necessary.

| <img width="1003" alt="screenshot 2019-02-21 at 17 57 48" src="https://user-images.githubusercontent.com/18581859/53169108-6262fe00-3602-11e9-99be-af95caa63988.png"> | <img width="958" alt="screenshot 2019-02-21 at 17 57 55" src="https://user-images.githubusercontent.com/18581859/53169109-6262fe00-3602-11e9-8c44-a0044053f95a.png"> |
|------|------|
| <img width="972" alt="screenshot 2019-02-21 at 17 58 27" src="https://user-images.githubusercontent.com/18581859/53169110-62fb9480-3602-11e9-89e8-c27c0add3a0f.png"> | <img width="969" alt="screenshot 2019-02-21 at 17 58 32" src="https://user-images.githubusercontent.com/18581859/53169112-62fb9480-3602-11e9-95a8-2432db55123c.png"> |

Normally Twenty Nineteen bugs would go to core Trac, but since this happens only on WordPress.com sites, I guess we could move this issue to `Automattic/themes`?

**Comment by sophiegyo** [comment]
Might be worth it :) I didn't really have the time to test with a different theme, it might be worth checking the new free Business themes too.

**Comment by sophiegyo** [comment]
So changing theme on the existing site has the same result for me:
https://cld.wthms.co/dYMPTV

After I changed the theme, I created a new page just in case, with three more pullqotes.
Process:
* Open new page
* Add three pullquotes
* Set different background colour for each
* Publish
* Visit page

Result:
https://cld.wthms.co/T5x6wW
Three monochrome pullquotes, despite the CSS.
This was directly after this page was published, there was no editing/updating.

Theme: Modern Business

**Comment by sophiegyo** [comment]
Note: Changing the theme to just Business (https://wordpress.com/theme/business/gutenlumos.wordpress.com) made the new page correctly display the pullquote colours.

I tested all the other Business themes (Modern, Sophisticated, Calm etc) and each one of them, along with Twenty Nineteen, has this issue.

**Comment by mmtr** [comment]
Seems to be solved after removing Gutenlypso in favor of Gutenframe (#31196).
